### DISCUSSION TITLE
CS%20Meng%40Cornell%20Tech/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#66] [Announcements]
# CS%20Meng%40Cornell%20Tech/

https://csmsapp.github.io/CS%20Meng%40Cornell%20Tech/

**Comment by b03505036** [comment]
这CS项目的bar其实不在于创业，身边同学录取的其实没人创业。
大多数都是陆本名校 & 海本名校，美本相对宽松
陆本同学：清北复交浙大武大中山
海本同学：台大，台清华，台交大，港三，滑铁卢，多大，墨尔本等等

上面根本没人创过业，但GPA:3.8 up & 一小段实习

**Reply by CindyCindy424** [reply]
刚拿到offer 想请教一下这个项目可以想办法延期吗？

**Comment by Geeksongs** [comment]
有大佬知道伊萨卡 cs meng 是啥项目啊？这个链接里似乎没有，https://tech.cornell.edu/programs/masters-programs/
